platform/kobo/koreader.sh

```diff
@@ -249,11 +258,13 @@ fi
 CRASH_COUNT=0
 CRASH_TS=0
 CRASH_PREV_TS=0
+# List of supported special return codes
+KO_RC_RESTART=85
+KO_RC_USBMS=86
```

Comment:
Is that the same as Cervantes? We'd probably want to keep them roughly the same if not. ;-)

Reply:
Yep!